### DISCUSSION TITLE
NPM Not Found

### DISCUSSION THREAD

**Discussion by HunterJuneau** [nss-evening-cohort-14/evening-client-side#229] [Help Me!] [answered]
## Your Name

Hunter Juneau

## Describe your issue

When I try to `npm i` it gives me this error: `/mnt/c/Users/huntl/AppData/Roaming/npm/npm: 12: node: not found`

## What have you tried?

I looked up how to install NPM and it says it comes with NodeJS. I tried to check my version of NodeJS with `node -v` and it says:
```
Command 'node' not found, but can be installed with:
sudo apt install nodejs
```
When running the above command, it says:
```
Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package nodejs
```
I tried to install NodeJS again but it didn't fix the issue.

**Comment by drteresavasquez** [comment, marked as answer]
Try Uninstalling it if you can and download it from the node js website. Make sure it is the LTS version.

**Reply by HunterJuneau** [reply]
No change.

**Reply by drteresavasquez** [reply]
I know you've run node --version right? Any results?

**Reply by HunterJuneau** [reply]
Yes, it says:
```
Command 'node' not found, but can be installed with:
sudo apt install nodejs
```

**Reply by drteresavasquez** [reply]
I mentioned this in study group, but let's jump on in class tomorrow to see what the issue is and we can report the resolution when we find it together.
